ledger/acctonline.go

```diff
@@ -815,13 +820,17 @@ func (ao *onlineAccounts) TopOnlineAccounts(rnd basics.Round, voteRnd basics.Rou
 			var accts map[basics.Address]*ledgercore.OnlineAccount
 			start := time.Now()
 			ledgerAccountsonlinetopCount.Inc(nil)
-			err = ao.dbs.Snapshot(func(ctx context.Context, tx *sql.Tx) (err error) {
-				arw := store.NewAccountsSQLReaderWriter(tx)
-				accts, err = arw.AccountsOnlineTop(rnd, batchOffset, batchSize, genesisProto)
+			err = ao.dbs.Snapshot(func(ctx context.Context, tx store.SnapshotScope) (err error) {
+				ar, err := tx.MakeAccountsReader()
```

Comment:
this looks not right. MakeAccountsReader calls AccountsInitDbQueries that prepares all queries. but this commit method needs only two.
This refactoring removes all benefits from having prepared queries and actually makes things worse from performance standpoint zero prepare vs ten prepare on each call

Reply:
Resolving this - there are two MakeAccountsReader implementations, one with all prepared and other with lazy. looks ok.